react/features/dropbox/functions.native.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+
+const { Dropbox } = NativeModules;
+
+/**
+ * Returns the display name for the current dropbox account.
+ *
+ * @param {string} token - The dropbox access token.
+ * @returns {Promise<string>}
+ */
+export function getDisplayName(token: string) {
+    return Dropbox.getDisplayName(token);
+}
+
+/**
+ * Returns information about the space usage for the current dropbox account.
+ *
+ * @param {string} token - The dropbox access token.
+ * @returns {Promise<Object>}
```

Comment:
Can you be more specific on what is returned by the promise as it's not so obvious here ? Maybe define a type and use it as the promise template ?